build.gradle

```diff
@@ -99,7 +100,7 @@ dependencyCheck {
 }
 
 processResources {
-    expand([transformApiVersion: "0.1.0"])
+    expand([transformApiVersion: "${transformApiVersion}"])
```

Comment:
I heroed this by deploying locally and confirming that the Accept-Version sent in the transform requests is 0.1.2.